### ISSUE TITLE
Using the WYSIWYG editor on mobile causes consistent jump to top behaviour

### ISSUE THREAD

**Issue by JakesRassie** [nhn/tui.editor#1083]
## Describe the bug
On mobile (Android Firefox) when typing in the WYSIWYG editor it jumps to the top of the editor (See video).

The scroll position is adjusted when the editor gains focus.

## To Reproduce
1. Open the SSE on Android in Firefox.
1. Focus at a section in the editor lower down on the page
1. The keyboard opens and the editor scrolls to the top

## Expected behavior
When typing in the WYSIWYG editor, the cursor should keep its position and the cursor position should not scroll out of view.

## Preview
![2020_07_08_12_12_40 2020-07-08 12_16_33](https://user-images.githubusercontent.com/14160322/86907172-edb83300-c114-11ea-836b-03ca1860455e.gif)

## Smartphone (please complete the following information):
 - Device: N/A
 - OS: Android
 - Browser: Firefox
 - Version: 68.10.1 

## Additional context

Link for replicating the bug: https://ndu68.csb.app/

No further investigation has been done but I have a hunch that this bug might be caused bu the way ToastUI determines the `cursorHeight` and `editorHeight` in the [`scrollIntoCursor()` function](https://github.com/nhn/tui.editor/blob/master/apps/editor/src/js/wysiwygEditor.js#L1032) when the keyboard is open.

**Comment by yar2001** [comment]
Yes! In android, the editor jump to top. While in IOS, the editor fail to show the cursor. Do you have any measure to solve it?
